masonry/src/testing/screenshots.rs

```diff
@@ -6,6 +6,38 @@
 use image::{GenericImageView as _, RgbImage};
 use nv_flip::{FlipImageRgb8, DEFAULT_PIXELS_PER_DEGREE};
 
+#[cfg(docsrs)]
+#[doc(hidden)]
+#[macro_export]
+macro_rules! include_screenshot {
+    ($path:literal $(, $caption:literal)? $(,)?) => {
+        concat!(
+            "![", $($caption,)? "]",
+            "(", "https://media.githubusercontent.com/media/linebender/xilem/",
+            "masonry-v", env!("CARGO_PKG_VERSION"), "/masonry/src/", $path,
+            ")",
+        )
+    };
+}
+
+// TODO:
+// - Check whether the docs.rs trick actually works.
+// - Check whether this method will work with cargo rdme.
```

Comment:
So this is a slightly messy one. The short answer is that it won't, and the long answer is that you can make it work, but it's messy, and the even longer answer is that even after making it work, I don't know if it will work when published to crates.io.

Either way, I'm not sure that this comment as-is is useful in this PR. Maybe there should be a comment at `/src/lib.rs`. Something like: `// TODO: Add screenshot. This can't use include_screenshot as that doesn't work with cargo-rdme`